collections/ansible_collections/cloudkit/templates/roles/ocp_virt_vm/meta/argument_specs.yaml

```diff
@@ -0,0 +1,142 @@
+argument_specs:
+  main:
+    options:
+      cluster_order:
+        type: dict
+        required: true
+        description: The cluster order containing VM configuration
+      cluster_working_namespace:
+        type: str
+        required: true
+        description: The namespace where VM resources will be created
+      template_parameters:
```

Comment:
I think it was discussed in another closed PR, but I think this list must be cleaned up